spec/requests/alert_definitions_spec.rb

```diff
@@ -306,4 +306,83 @@
     expect(alert_definition_profiles.first.reload.description).to eq("Updated Test Alert Profile 1")
     expect(alert_definition_profiles.second.reload.description).to eq("Updated Test Alert Profile 2")
   end
+
+  it "assigns alert definitions to a profile" do
+    api_basic_authorize collection_action_identifier(:alert_definition_profiles, :edit)
+    alert_definition_profile = FactoryGirl.create(:miq_alert_set)
+    alert_definitions = FactoryGirl.create_list(:miq_alert, 2)
+
+    post(api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile), :params => gen_request(:assign, alert_definitions))
```

Comment:
If I understand this correctly, this request provides more information than is sufficient.

```ruby
post(
  api_alert_definition_profile_alert_definitions_url(nil, alert_definition_profile),
  :params => {
    :action => "assign",
    :resources => [
      {:id => alert_definitions[0].id},
      {:id => alert_definitions[1].id}
    ]
  }
)
```

...I think would suffice